react/SliderLayout.tsx

```diff
@@ -54,10 +55,12 @@ function SliderLayout({
         totalItems={totalSlides}
         itemsPerPage={responsiveItemsPerPage}
         centerMode={responsiveCenterMode}
+        centerModeSlidesGap={centerModeSlidesGap}
```

Comment:
I don't think this value needs to be in the `SliderContext`. Any reason to add it there?

Reply:
Yes, i use it to calculate slideWidth and transformMap, see lines 252-255 and 289-294

Reply:
Nice! Missed that!